card/db/migrate_core_cards/20161103154836_update_keys.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+class UpdateKeys < Card::Migration::Core
+  def up
+    Card.pluck(:id, :name, :key).each do |id, name, key|
```

Comment:
Might be worth testing this on wikirate.  If this is fast enough without batches on wikirate, it should be fast enough anywhere.